Update snowflake-data-share.md (#182)
amplitude/amplitude-dev-center: docs/data/destinations/snowflake-data-share.md

Snowflake only supports data share within same region and same cloud. Amplitude's Snowflake is in US West (Oregon) region and using Amazon Web Services. To enable cross region cross cloud data share, reach out to your Account Manager at Amplitude or contact Amplitude Support.

> Only one Snowflake Data Share can be set up per project for Events Queries and Merge User Queries.

Reviewer: ```suggestion
Only one Snowflake Data Share can be set up per project for Events Queries and Merge User Queries.

Amplitude supports only one Snowflake Data Share per project for events and merge user queries.
```

> Only one Snowflake Data Share can be set up per project for Events Queries and Merge User Queries.

Reviewer: ```suggestion
Amplitude supports only one Snowflake Data Share per project for events and merge user queries.
```
